cmd/kubermatic-api/swagger.json

```diff
@@ -28470,7 +28470,7 @@
           "x-go-name": "PodAntiAffinityPreset"
         },
         "primaryDiskOSImage": {
-          "description": "PrimaryDiskOSImage states the source from which the imported image will be downloaded.",
+          "description": "PrimaryDiskOSImage states the source from which the imported image will be downloaded.\nFor now (might change later on) the Os Image contains:\na URL.\na DataVolume Name (not prefixed with namespace/).\nThe DataVolume name will be prefixed with the dedicated infra cluster namespace name (cluster-xyz/datavolumename)\nbefore storage in the provider config. It is then splitted in machine-controller to get the namespace and the name\nof the DataVolume to clone from.\nWe keep for now this mechanism to be able to clone the Data Volume in the future from different namespaces.",
```

Comment:
I would remove this heads-up about temporary change
> For now (might change later on) the Os Image contains:\na URL.\na DataVolume Name (not prefixed with namespace/).\nThe DataVolume name will be prefixed with the dedicated infra cluster namespace name (cluster-xyz/datavolumename)

The plan is that we will have a drop down field for the UI so in the future users won't have to type anything in there. I think it's just ok now to say that over this field we download an image from http source or we use data volume cloning.

Reply:
Good point, I've updated accordingly.